src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -206,6 +205,25 @@ ManifestCache::configManifest (
     }
 }
 
+boost::optional<std::string>
+ManifestCache::member (PublicKey const& identity) const
+{
+    std::lock_guard<std::mutex> lock (mutex_);
+
+    auto ret = map_.find (identity);
+
+    if (ret != map_.end())
+        return ret->second.comment;
+
+    return boost::none;
+}
+
+bool
+ManifestCache::trusted (PublicKey const& identity) const
+{
+    return static_cast<bool> (member(identity));
```

Comment:
I was using `ValidatorList` as an example
https://github.com/ripple/rippled/blob/30eff2b5208351c6ece0274f466d34c2a7c30f75/src/ripple/app/misc/impl/ValidatorList.cpp#L33-L54

I'll see if I can make those changes in both classes.